media/css/m24/components/footer-refresh.scss

```diff
@@ -341,52 +294,51 @@ $max-footer-content-width: $content-max;
     border-color: transparent;
 
     &::before {
-        display: block;
-        content: "";
-        position: absolute;
-        width: 32px;
-        height: 32px;
         @include bidi(((left, 0, 0), (right, 0, 102px)));
-        bottom: 0;
+        background: transparent url('/media/img/icons/m24-small/globe-white.svg') center center no-repeat;
+        background-size: 18px 18px;
+        top: 50%;
+        margin-top: -16px;
         color: $m24-color-white;
-        background-repeat: no-repeat;
-        background-position-x: 8px;
-        background-position-y: -2px;
-        background-image: url('/media/img/logos/mozilla/footer-refresh/icon-globe.svg');
+        content: "";
+        display: block;
+        height: 32px;
+        width: 32px;
+        position: absolute;
     }
 
     label {
-        border: 0;
-        clip: rect(0 0 0 0);
-        height: 1px;
-        margin: -1px;
-        overflow: hidden;
-        padding: 0;
-        position: absolute;
-        width: 1px;
+        @include visually-hidden;
     }
 
     .mzp-c-language-switcher-link {
         @include visually-hidden;
+
+        &:focus,
+        &:focus-visible {
```

Comment:
**Praise:** Yay for visible focus.

Reply:
> "Made the language link visible on focus (I was confused when tabbing through and it took me a while to realize the link was there)."

It's actually in a weird order — being in between the label and its select. Would it break anything if moved after the select (and also change its text to something like "All languages"), hopefully in preparation for some nonJS fallback for the component? (Now the link is actually hidden even in nonJS, where it would be beneficial instead of the form components without a way to submit — but it's still a GET form with ?lang=foo values so adding that back one day should be viable too, the logic to turn [`/th/?lang=vi` to `/vi`](https://www-demo4.allizom.org/th/?lang=vi) is there.)

Reply:
Yeah I think this whole widget needs some refactoring. I think at the very least we can move the link to before the label and change the link text (that'll be a new string).

Reply:
I know it's a burden to add a new string, but the two repetitive items, with the same text, serving two different interactions, might be confusing.  (I tried to look what strings are available in general, and haven't found anything readily available unfortunately.)